charts/kargo/templates/external-webhooks-server/cluster-role.yaml

```diff
@@ -7,6 +7,22 @@ metadata:
     {{- include "kargo.labels" . | nindent 4 }}
     {{- include "kargo.externalWebhooksServer.labels" . | nindent 4 }}
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - kargo.akuity.io
+  resources:
+  - projectconfigs
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Do we additionally need permission to patch the `projectconfigs/statsus` subresources?

Reply:
Nevermind. Only the management controller needs that. Don't mind me.